scripts/linters/pylint_extensions.py

```diff
@@ -43,6 +45,14 @@
 ALLOWED_PRAGMAS_FOR_INLINE_COMMENTS = [
     'pylint:', 'isort:', 'type: ignore', 'pragma:', 'https:']
 
+ALLOWED_LINES_OF_GAP_IN_COMMENT = 15
+
+EXCLUDED_TYPE_COMMENT_DIRECTORIES = [
+    'core/controllers/',
+    'extensions/',
+    'scripts/'
+]
```

Comment:
What is the plan for these? Should we aim to eventually add type comments to them, or leave them in the exclusion list indefinitely?

Reply:
I think they'll be covered in Milestone 2.

But @sahiljoster32 you should leave a TODO here, please.

Reply:
Added!